clusterloader2/api/validation/validation.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+// VerifyConfig does a simple validation for config and all included structures.
+// Returns list of all not satisfied conditions.
+func VerifyConfig(c *api.Config) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if c.AutomanagedNamespaces < 0 {
+		allErrs = append(allErrs, field.Invalid(nil, "automanagedNamespaces", "must be non-negative"))
```

Comment:
Also, the second argument to field.Invalid is the actual value of the field, not it's name - please fix here and in a number of toher places.

Reply:
For some reason, I though that requires (path to structure, field name, text). I stand corrected.
Fixing other places.